Dockerfile

```diff
@@ -1,13 +1,5 @@
-FROM navikt/node-express:18
-
-# Workaround: får tilfeldige feil ved npm install uten neste linje
-RUN npm config set unsafe-perm=true 
-
+FROM gcr.io/distroless/nodejs20-debian12
```

Comment:
nodejs lts er 22.
Er appen vår ikke kompatibel med det? 
Kunne man hatt noe sånn :latest eller lignende isåfall for å slippe å bumpe versjon?

Reply:
Jeg vil helst unngå å stole på at "latest" alltid vil fungere for oss, men å oppgradere til versjon 22 er jeg med på!